sdk/communication/Azure.Communication.CallingServer/tests/CallRecording/ContentDownloadLiveTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Communication.CallingServer.Tests.Infrastructure;
+using NUnit.Framework;
+
+namespace Azure.Communication.CallingServer
+{
+    public class ContentDownloadLiveTests : CallAutomationClientLiveTestsBase
+    {
+        public ContentDownloadLiveTests(bool isAsync) : base(isAsync)
+        { }
+
+        [Test]
+        public async Task DownloadMetadata()
+        {
+            if (SkipCallingServerInteractionLiveTests)
+            {
+                Assert.Ignore("Skip callingserver interaction live tests flag is on");
+            }
+
+            CallAutomationClient client = CreateInstrumentedCallingServerClientWithConnectionString();
+            string documentId = "0-wus-d4-d4c223871b58d3664401b66d35fca784";
+            Uri metadataEndpoint = new Uri($"https://us-storage.asm.skype.com/v1/objects/{documentId}/content/acsmetadata");
+            CallRecording callRecordingClient = client.GetCallRecording();
+            Stream metadata = await callRecordingClient.DownloadStreamingAsync(metadataEndpoint);
+            Assert.IsNotNull(metadata);
+            JsonElement expected = JsonDocument.Parse(metadata).RootElement;
```

Comment:
nit since we got the metadata from calling the DownloadStreamAsync method, should the variable name be "actual" instead of "expected"?